packages/teleterm/src/ui/services/docs/docs.ts

```diff
@@ -136,6 +135,13 @@ export default class DocumentService extends Store<State> {
     });
   }
 
+  addAndOpen(doc: Document) {
```

Comment:
I added a new method because invoking `add()` and `open()` separately caused an issue in async handler (`fit()` in xterm didn't take up all the container space). Invoking these methods works correctly in non-async flows because React batches state updates and new document is added and shown at the same time.

Reply:
Removed